.github/workflows/python-package.yml

```diff
@@ -52,7 +52,28 @@ jobs:
     - name: Test Wheel
       run: |
         python run_tests.py
-    
+  
+  build_arm64:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+          python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up QEMU
+      id: qemu
+      uses: docker/setup-qemu-action@v1  
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Build and Test Wheel Arm64
+      run: |
+        sudo chmod +x run_manylinux_aarch64_build.sh
```

Comment:
can we remove the sudo?

Reply:
> can we remove the sudo?

Now run_manylinux_aarch64_build.sh is not getting used. So, this is not required.